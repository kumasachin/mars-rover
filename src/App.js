import React, { useState } from "react";
import {
  Planet,
} from "./components/";
import "./App.css";

const App = () => {
  return (
    <div className="App">
      <Planet name="Mars" />
    </div>
  );
}

export default App;
