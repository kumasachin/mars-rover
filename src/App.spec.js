import { render, screen, fireEvent, waitFor, cleanup } from '@testing-library/react';
import App from './App';

describe("App", () => {
  afterEach(cleanup);
  beforeEach(()=> render(<App />));
  it('should take a snapshot', () => {
    const { asFragment } = render(<App />);
    expect(asFragment(<App />)).toMatchSnapshot()
  });
});
