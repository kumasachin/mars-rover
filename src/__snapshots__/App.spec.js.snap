// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should take a snapshot 1`] = `
<DocumentFragment>
  <div
    class="App"
  >
    <div>
      Loading...
    </div>
  </div>
</DocumentFragment>
`;
